lib/rules/selector-class-pattern/__tests__/index.js

```diff
@@ -141,9 +149,11 @@ testRule({
 	reject: [
 		{
 			code: '.A { &__B { }}',
-			message: messages.expected('A__B', /^[A-Z]+$/),
+			message: messages.expected('.A__B', /^[A-Z]+$/),
 			line: 1,
 			column: 6,
+			endLine: 1,
+			endColumn: 11,
```

Comment:
[note] Correctly, `endColumn` should be `10` but actually `11`:

```scss
.A { &__B { }}
/*   ↑   ↑
     6   10
*/
```

`11` is calculated from `6 + ".A__B".length` (`".A__B"` is a resolved selector).

For details, see the note in the source code:
https://github.com/stylelint/stylelint/pull/6227/files#diff-3cdac1b62a3c129821ddbefec56839ea6af7b04c9b78a7c3f4ea5054197bc74fR92